Fix:
error: implicit declaration of function 'conf_new_patternfile' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
https://github.com/jordansissel/grok/pull/40
error: implicit declaration of function 'yylex' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
https://github.com/jordansissel/grok/issues/39
--- grok_config.h.orig	2011-10-28 02:17:11.000000000 -0500
+++ grok_config.h	2022-01-17 03:06:49.000000000 -0600
@@ -21,6 +21,7 @@
 
 void conf_init(struct config *conf);
 void conf_new_program(struct config *conf);
+void conf_new_patternfile(struct config *conf);
 void conf_new_input(struct config *conf);
 void conf_new_input_process(struct config *conf, char *cmd);
 void conf_new_input_file(struct config *conf, char *filename);
--- conf.y.orig	2011-10-28 02:17:11.000000000 -0500
+++ conf.y	2022-01-20 12:46:24.000000000 -0600
@@ -12,6 +12,8 @@
   fprintf (stderr, "Syntax error: %s\n", s);
 }
 
+int yylex (YYSTYPE *lvalp, YYLTYPE *llocp);
+
 #define DEBUGMASK(val) ((val > 0) ? ~0 : 0)
 %}
 
